test/cli/autocorrect-requires-ancestor-block/autocorrect-requires-ancestor-block.rb

```diff
@@ -0,0 +1,14 @@
+# typed: true
+
+module RA1; end
+module RA2; end
+class C; end
+
+class A
+  extend T::Helpers
+
+  abstract!
+
+  requires_ancestor C
+  requires_ancestor RA1, RA2
+end
```

Comment:
This program will raise errors even after the autocorrects have been applied, would it be worth making `A` a module instead?

Reply:
Switched to a module 👍